site/gatsby-site/src/templates/backups.js

```diff
@@ -15,6 +15,22 @@ const Backups = ({ pageContext, ...props }) => {
     return null;
   }
 
+  const parseCreationDate = (key) => {
```

Comment:
Please provide a comment demonstrating the expected input format/example input.

Reply:
Done!